docs/UserGuide.md

```diff
@@ -183,152 +193,215 @@ Adds an event with specific starting and ending date or time to TaskTracker.
   date.
 
   The default year of `dd/mm` is the current year and the default month for
-  `dd` is current month.
+  `dd` is the current month.
 
-* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCD` which
-  each letter represents a number ranges from 0 to 9. `AB` represent the hour
-  in 24 hour system, while `CD` represent the minute. Letter `A` can be omitted
-  if it is `0`.
+* `STARTING_DATE` and `STARTING_TIME` must follow fixed formats : `ABCDam` or `ABCDpm`
+  where each letter represents a digit. `AB` represent the hour
```

Comment:
How about rewording it as `AB represents the hour (1-12), while CD represents the minute (00-60)`?